hoodie-client/src/main/java/com/uber/hoodie/table/HoodieCopyOnWriteTable.java

```diff
@@ -518,6 +515,57 @@ protected HoodieMergeHandle getUpdateHandle(String commitTime, String fileLoc, I
         }
     }
 
+    protected Map<FileStatus, Boolean> delete(String partitionPath, List<String> commits) throws IOException {
```

Comment:
why call this `delete` instead of `clean`?

Reply:
Since there can be > 1 type of action in MOR (deltacommit/compaction), I parallelize commits in the `rollback(...)` method of HoodieMergeOnReadTable, hence can't use HoodieCopyOnWriteTable `rollback(...)` and had to separate out the `delete()` method.
`clean` is used for a different purpose right ? Clean triggers a list of files to figure out if clean policy is matched etc, this delete(..) is to delete files whose commitTime fall in the list of commits passed.

Reply:
okay got it.. this just separates out the actual deletes..